src/legacy/ui/public/_index.scss

```diff
@@ -11,5 +11,4 @@
 @import './accessibility/index';
 @import './directives/index';
 @import './error_url_overflow/index';
-@import './field_editor/index';
-@import '../../../plugins/management/public/components/index';
+@import './exit_full_screen/index';
```

Comment:
I think this was a bad merge with master? #63443 removes this import from this file.

Reply:
removed